config_sections.go

```diff
@@ -63,6 +64,33 @@ func NewConfigSections() ConfigSections {
 }
 
 func (c *ConfigSections) populateSections(ctx context.Context) error {
+	if err := c.getSSMParameters(ctx); err != nil && !errors.Is(err, ssmDisabledErr) {
```

Comment:
Is there a concern that this will use too many API calls if it's used for every `evergreen.GetConfig`? In the [scope](https://docs.google.com/document/d/1dUIZRMVF2rdsfVIZ1TDF7W3SOBuzenBFTCWqWp_wbts/edit#heading=h.25bcvwewueb7) for using Parameter Store, there was a mention that we may need to cache values because reading them dynamically each time from Parameter Store incurs a cost per read. Given how often we call `evergreen.GetConfig`, I imagine this would start stressing any rate limits/read costs that Parameter Store has if we do it as a drop-in replacement for the DB call.

Reply:
This is a good point. 😎 
I think it's been addressed.